### PR TITLE
open-cluster-management/backlog#10824

### PR DESCRIPTION
**Pull request by swopebe**
--argo integrated into console and what's new

### CODE REVIEW

File: applications/app_console.adoc [hunk 1]
```diff
@@ -46,18 +46,21 @@ Click on an application name in the table to view details about a single applica
 * Cluster details, such as resource status.
 * Subscription details
 * Resource topology
+* Actions such as *Launch Argo editor*
 
-Click the `Editor` tab to edit your application and related resources.
+Click the `Editor` tab to edit your application and related resources. //???
```
Comment (swopebe) [inline applications/app_console.adoc:52]: please look at this one, as well.

Comment (birsanv) [reply applications/app_console.adoc:52]: Brandi, the editor tab shows only for ACM apps
You can say : The Editor tab is only available for ACM applications. Click the `Editor` tab to edit your application and related resources.

File: applications/app_console.adoc [hunk 2]
```diff
@@ -46,18 +46,21 @@ Click on an application name in the table to view details about a single applica
...
 == Resource topology
 
 The topology provides a visual representation of the application that was selected including the resources deployed by this application on target clusters.
 
-* You can select any component from the topology view to view more details.
+* You can select any component from the Topology view to view more details.
 
 * View the deployment details for any resource deployed by this application by clicking on the resource node to open the properties view.
 
+* View your Argo applications and launch to the Argo editor.
```
Comment (birsanv) [inline applications/app_console.adoc]: View your Argo applications and launch to the Argo editor, **if selected application is Argo.**

**Review by birsanv — COMMENTED** [review]
I have a few comments, everything else looks good, thanks

File: applications/app_console.adoc (hunk 1 above)
Comment (swopebe) [reply applications/app_console.adoc:52]: Thanks! That explains why I didn't see it. Will change.

Comment (swopebe) [reply applications/app_console.adoc:52]: this is changed

**Review by birsanv — APPROVED** [review]
lgtm

**Pull request merged**